src/scripts/main.js

```diff
@@ -1 +1,77 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve();
+    }
+  });
```

Comment:
Consider removing the event listener after the promise is resolved or rejected to prevent memory leaks or unexpected behavior.